main/index.js

```diff
@@ -14,6 +14,7 @@ const { setupAppMenu } = require('./app-menu')
 
 const { ActivityLog } = require('./activity-log')
 const { ipcMain } = require('electron/main')
+const { JobCounter } = require('./job-counter')
```

Comment:
What do you think about calling this `JobStats` instead? This way we can add additional stats in the future (like job execution time or error rate) and don't have the awkwardly similar but different events `JOB_COUNTER_UPDATED` and `job-count-updated`.

Reply:
Also it makes more sense to me to call this stat something like `totalJobsCompleted` or `jobsCompleted`, because we're counting the completed jobs and not the overall jobs (technically). This way UI and variables are aligned.

Reply:
> What do you think about calling this JobStats instead?

I love that idea! 💯

Reply:
> call this stat something like `totalJobsCompleted`

🎱